### DISCUSSION TITLE
System / Mail / Error on Test Email Button

### DISCUSSION THREAD

**Discussion by ThatRobbo** [requarks/wiki#4937] [Error / Bug Report]
Hi,
When i try to setup mail and send a test email (via Office 365) I get the following error when i clikc the send email button:

140231884557104:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:331:

Am i missing something as the settings appear to be right, the only thing i can think of is that I'm using a custom domain and i haven't secured it with an SSL certificate yet. I've put my system info at the bottom if that helps?

Thanks in advance.
Dan

![Config](https://user-images.githubusercontent.com/98412056/151015569-698136fd-5468-456c-86d5-5d9c1b61eaec.png)

**Comment by dcpc007** [comment]
I just configure my email with a gmail account (and password) and got the same error.
Now i change the port and smtp for another ones in the google list. And no more error. But i don't get the mail in test after. (trying to find a log to debug)

![image](https://user-images.githubusercontent.com/847516/151025386-f8bb85dc-274b-4e65-8796-ccaad1c3e00f.png)
for my current config.

**Comment by DarkArc** [comment]
I have the same issue trying to use mailtrap (ports 587, 25, and 2525 all do not work).

**Comment by DarkArc** [comment]
Duplicate of https://github.com/requarks/wiki/discussions/3574
